packages/core/src/components/ListContainer/ListItem/ListItem.styles.tsx

```diff
@@ -0,0 +1,95 @@
+import styled from "@emotion/styled";
+import { theme } from "@hitachivantara/uikit-styles";
+import { transientOptions } from "utils/transientOptions";
+
+export const StyledListItem = styled(
+  "li",
+  transientOptions
+)(
+  ({
+    $gutters,
+    $interactive,
+    $selected,
+    $disabled,
+    $startAdornment,
+    $endAdornment,
+  }: {
+    $gutters: boolean;
+    $interactive: boolean;
+    $selected: boolean;
+    $disabled: boolean;
+    $startAdornment: boolean;
+    $endAdornment: boolean;
+  }) => ({
+    color: theme.colors.acce1,
+    fontSize: "12px",
+    letterSpacing: "0.02em",
+    fontWeight: 400,
+    padding: 0,
+    display: "block",
+    height: "32px",
+    lineHeight: "32px",
+    listStyleType: "none",
+    overflow: "hidden",
+    textOverflow: "ellipsis",
+    whiteSpace: "nowrap",
+    position: "relative",
+
+    "&:not(:last-child)": {
+      marginBottom: "8px",
+    },
+    "&.condensed": {
+      marginBottom: 0,
+    },
+
+    ...($gutters && {
+      padding: `0 ${theme.spacing(1)}`,
+
+      "&.withStartAdornment": {
+        paddingLeft: 0,
+      },
+      "&.withEndAdornment": {
+        paddingRight: 0,
+      },
+    }),
+    ...($interactive && {
+      cursor: "pointer",
+      "&:not(.disabled):not(.selected):hover": {
+        backgroundColor: theme.list.hoverColor,
+      },
+      "&.disabled": {
+        cursor: "not-allowed",
+      },
+    }),
+    ...($selected && {
+      backgroundColor: theme.list.hoverColor,
+    }),
+    ...($disabled && {
+      color: theme.colors.atmo5,
+    }),
+    ...($startAdornment && {
+      "& > div": {
+        float: "left",
+      },
+
+      "& svg": {
+        boxShadow: "none !important",
+        outline: "none !important",
+      },
+      ".disabled>& svg *.color0": {
```

Comment:
The fill color is not changing when the item is disabled. I think the "disabled" class is not being attached to the start adornment so this selector is not working. The same is happening for the end adornment.